### PR TITLE
Add Kotlin to Barista! :tada:

### PR DESCRIPTION
**Pull request by SmasSive**
Welcome to the futu... Oh wait! To the presenter! 🍷

### CODE REVIEW

File: sample/build.gradle [hunk 1]
```diff
@@ -25,8 +26,12 @@ dependencies {
...
 
+  compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+
   testCompile 'junit:junit:4.12'
 
   androidTestCompile project(':library')
-
+}
+repositories {
+  mavenCentral()
```
Comment (Sloy) [inline sample/build.gradle]: I don't think that adding mavenCentral() is needed for the subprojects. We already add jcenter() to all subprojects from the main build.gradle file.

File: build.gradle [hunk 2]
```diff
@@ -1,10 +1,12 @@
 buildscript {
-    repositories {
+  ext.kotlin_version = '1.1.4-2'
+  repositories {
```
Comment (Sloy) [inline build.gradle]: Did the indentation of `repositories` change? Is it wrong now or was it wrong before?

Comment (SmasSive) [inline build.gradle]: AFAIK it uses 4 spaces indentation while library uses 4 and sample uses 2. The code is indented to 2.
I didn't want to change too much lines but I think is necessary a little reformat on those two gradle files. What do you think?

File: sample/build.gradle (hunk 1 above)
Comment (SmasSive) [inline sample/build.gradle]: Yes, you are right!

**Comment by rocboronat** [comment]
BTW, this PR is related with #69

File: build.gradle (hunk 2 above)
Comment (Sloy) [reply build.gradle]: Ok let's not worry about it here

**Pull request merged**